qlib/data/ops.py

```diff
@@ -32,6 +32,91 @@
 
 np.seterr(invalid="ignore")
 
+
+#################### Change instrument ########################
```

Comment:
This duplicates with the docs string.
I think we can keep only the one in the docstring.

Reply:
Yes, sure. I wrote the first paragraph to motivate the class, and it should be fine to keep only those in the doc string and I'll update that.

Reply:
Hi, @wan9c9  did you fix this ? 
Thanks